app/controllers/admin/stats_controller.rb

```diff
@@ -29,10 +29,23 @@ def top_depositors_count
     end
 
     def document_by_permission
+      return document_by_date_by_permission if @users_stats[:file_start_date]
+
       files_count = {}
       files_count[:total] = GenericFile.count
-      files_count[:public] = GenericFile.where(Solrizer.solr_name('read_access_group', :symbol) => 'public').count
-      files_count[:registered] = GenericFile.where(Solrizer.solr_name('read_access_group', :symbol) =>'registered').count
+      files_count[:public] = GenericFile.where_public.count
+      files_count[:registered] = GenericFile.where_registered.count
+      files_count[:private] = files_count[:total] - (files_count[:registered] + files_count[:public])
```

Comment:
@mjgiarlo the negative outside the parens is the same as subtracting both.  I believe that the parens make more logical sense based on the fact that the private files is the total minus the total of other categories.  We can subtract both, but just removing the parens would cause a logic error.